### PR TITLE
[DO NOT MERGE] added validate_clusters true

### PR DESCRIPTION
**Pull request by ijsnellf**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: proxy/envoy/resources.go [hunk 1]
```diff
@@ -88,9 +88,9 @@ const (
 	RedisProxyFilter = "redis_proxy"
 
 	// RedisDefaultOpTimeout is the op timeout used for Redis Proxy filter
-	// Currently it is set to 30ms (conversion happens in the filter)
+	// Currently it is set to 30ms
 	// TODO - Allow this to be configured.
-	RedisDefaultOpTimeout = 30 * time.Second
+	RedisDefaultOpTimeout = 30 * time.Millisecond
```
Comment (ldemailly) [inline proxy/envoy/resources.go]: seems unrelated ? isn't this short ?

Comment (ijsnellf) [reply proxy/envoy/resources.go]: completely unrelated, just accidentally got swept into the commit

Comment (ijsnellf) [reply proxy/envoy/resources.go]: should not effect anything.

File: proxy/envoy/resources.go [hunk 2]
```diff
@@ -88,9 +88,9 @@ const (
 	RedisProxyFilter = "redis_proxy"
 
 	// RedisDefaultOpTimeout is the op timeout used for Redis Proxy filter
-	// Currently it is set to 30ms (conversion happens in the filter)
+	// Currently it is set to 30ms
```
Comment (andraxylia) [inline proxy/envoy/resources.go]: 30s is a more realistic timeout, isn't it?
let's update just the comment to say 30s and not change the timeout value.

Comment (ijsnellf) [reply proxy/envoy/resources.go]: yes, you are right. please ignore, though. it doesn't have anything to do with the rest of the PR. i'll clean it up

**Comment by ldemailly** [comment]
Can someone make that PR pass so I can try the Sha with the 404 test?

**Comment by ldemailly** [comment]
I think we should still explore this option, at least confirm that the PR was generating the right config and it was used by envoy before dropping it

**Comment by andraxylia** [comment]
Right, validate_cluster is still something that we want to add in master.
